crates/environ/src/cache/worker.rs

```diff
@@ -100,7 +100,7 @@ impl Worker {
         let sent_event = self.sender.try_send(event.clone());
 
         if let Err(ref err) = sent_event {
-            info!(
+            debug!(
```

Comment:
This error seems is about more than just "here's some stuff you may want if you're debugging". It's more "something unexpected happened which isn't immediately fatal but which may mean the module cache isn't actually caching things".

Reply:
this sounds like `warn` again

Reply:
Makes sense. Would you mind changing this one to `warn`?